modules/vardictjava/main.nf

```diff
@@ -23,8 +23,9 @@ process VARDICTJAVA {
     def args = task.ext.args ?: ''
     def args2 = task.ext.args2 ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-    def VERSION = '1.8.3' // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
+
     """
+    export JAVA_OPTS='"-Xms${task.memory.toMega()/4}m" "-Xmx${task.memory.toGiga()}g" "-Dsamjdk.reference_fasta=$fasta"'
     vardict-java \\
         $args \\
         -c 1 -S 2 -E 3 \\
```

Comment:
This line should be supplied via `$args`.

Reply:
Why should this line be supplied by `$args`?

Reply:
I was going to say this was dependent on the input, but I just realised this is standard to BAM. So ignore it.